game_engine/ecs/systems/ServerInputSystem.cpp

```
@@ -0,0 +1,87 @@
+#include "ServerInputSystem.hpp"
+#include "Events.hpp"
+#include "Macro.hpp"
+#include "Packet.hpp"
+#include "PlayerComponent.hpp"
+#include "PositionComponent.hpp"
+#include "SpeedComponent.hpp"
+
+void ecs::ServerInputSystem::update(float deltaTime) {
```

Clang (14.0.6)

[warning] method 'update' can be made static

(readability-convert-member-functions-to-static)